src/utils/validation/evaluation-administration-schema.ts

```diff
@@ -8,36 +8,30 @@ export const createEvaluationAdministrationSchema = object().shape({
       const end_date = this.parent.eval_period_end_date
       return new Date(start_date) <= new Date(end_date)
     })
-    .test(
-      "start-date",
-      "Must select a date that is within the valid range.",
```

Comment:
Don't remove this validation, this is for when range of date is more than or less than 50 years ago.

Reply:
Up